addons/point_of_sale/models/pos_payment_method.py

```diff
@@ -61,14 +61,16 @@ def _get_payment_method_type(self):
         help='Type of QR-code to be generated for this payment method.',
     )
     hide_qr_code_method = fields.Boolean(compute='_compute_hide_qr_code_method')
+    terminal_id = fields.Text()
```

Comment:
```suggestion
    terminal_identifier = fields.Text()
```

It's better to reserve the `_id` to a related m2o field.

Reply:
or `terminal_name`

Reply:
I see that you are now using `terminal_identifier`, however, I think you forgot to change other references.